src/frontends/pytorch/src/op/roll.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_roll(NodeContext& context) {
+    const auto data = context.get_input(0);
+    const auto shifts = context.get_input(1);
+    const auto axes = context.get_input(2);
+    // if axes was not set 
+    if (axes.get_shape() != shifts.get_shape()) {
```

Comment:
if `axes` is not provided what `axes.get_shape()` returns?

Reply:
If it is not provided, default it 0

Reply:
By this verification you restrict `axes` and `shifts` to have static shape, but I don't think this is required.

Reply:
I can add a check that also tests if both axes and shifts have dynamic shapes. Would be that sufficient?